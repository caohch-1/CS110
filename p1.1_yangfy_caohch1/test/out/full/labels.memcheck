==945== Memcheck, a memory error detector
==945== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==945== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==945== Command: ../assembler in/full/labels.s out/full/labels.int out/full/labels.symtbl out/full/labels.out -log out/full/labels.log
==945== 
==945== error calling PR_SET_PTRACER, vgdb might block
==945== 
==945== HEAP SUMMARY:
==945==     in use at exit: 0 bytes in 0 blocks
==945==   total heap usage: 24 allocs, 24 frees, 7,240 bytes allocated
==945== 
==945== All heap blocks were freed -- no leaks are possible
==945== 
==945== For lists of detected and suppressed errors, rerun with: -s
==945== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
