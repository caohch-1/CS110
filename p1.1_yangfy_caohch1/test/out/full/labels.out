.text
00a30067
01436293

.symbol
